guides/common/attributes.adoc

```diff
@@ -1,6 +1,6 @@
 // Document state: "nightly" for master, "stable" for last two releases,
 // "unsupported" for the rest and "satellite" for satellite build
-:DocState: unsupported
+:DocState: stable
```

Comment:
Thinking more about this, it's tricky. While Foreman 3.6 is going GA, Katello has yet to release RC1. I'm going to revert this bit.